Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -41,6 +41,7 @@ def setUp(self):
             self.scope = config["scope"]
             self.subscription_id = config["subscription_id"]
             self.container_name = config["container_name"]
+            self.name_prefix = config["producer_fun_prefix"]
```

Comment:
Consider using 'producer_prefix' instead of 'name_prefix'.

Reply:
done